image-create

```diff
@@ -56,7 +56,13 @@ class MachineBuilder:
     def __init__(self, machine):
         self.machine = machine
         self.iso = self.machine.image.endswith("boot")
-        self.suffix = ".iso" if self.iso else ".qcow2"
+        self.payload = self.machine.image.endswith("payload")
+        if self.iso:
+            self.suffix = ".iso"
+        elif self.payload:
+            self.suffix = ".tar.gz"
+        else:
+            self.suffix = ".qcow2"
```

Comment:
Is this OK in terms of cockpit infra? I find deciding based on name a bit hacky, but don't know how correct this needs to be.

Reply:
it's fine for now.